recipes/cunit/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "2.1.3":
+    url: "http://downloads.sourceforge.net/cunit/CUnit-2.1-3.tar.bz2"
```

Comment:
Was it on purpose to change `2.1-3` to `2.1.3`?

Reply:
Oversight. Thanks

Reply:
Oh it is already merged.
What to do now?

Reply:
Hmmm 

Since we can't de-publish from the Conan remote..

Either we leave it and adapt the correct version for upcoming ones or we remove `2.1.3` from git only and add the correct one

Not sure what is better

Reply:
Future versions aren't very likely. The project was forked and lives [here](https://gitlab.com/cunity/cunit). None of the original authors actually confirmed/transferred, so I'm unsure whether anyone should switch. Anyway, thought I'd mention it.

Reply:
Thanks. I didn't know about the fork!

Reply:
😖  I think `2.1-3` makes a lot of sense, as it is the last _official version_. Some projects might be using that version instead of the fork.

I would say that we need `2.1-3` and I would remove `2.1.3` from this repo, but I know other @conan-io/barbarians  won't agree with me. Let's see what they have to say.

Reply:
As we just uploaded, I see no problem removing, but we have to be quickly.